src/rdkafka_conf.c

```diff
@@ -688,6 +697,18 @@ static const struct rd_kafka_property rd_kafka_properties[] = {
 	{ _RK_GLOBAL|_RK_HIGH, "sasl.password", _RK_C_STR,
 	  _RK(sasl.password),
 	  "SASL password for use with the PLAIN and SASL-SCRAM-.. mechanism" },
+#if WITH_SASL_OAUTHBEARER
+	{ _RK_GLOBAL, "sasl.oauthbearer.config", _RK_C_STR,
+	  _RK(sasl.oauthbearer_config),
+	  "SASL/OAUTHBEARER configuration. The format is implementation-dependent and must be parsed accordingly. "
+          "The default unsecured token implementation recognizes space-separated name=value pairs with valid names including "
```

Comment:
maybe call out the name or acronym of unsecured JWT so people know what to google for

Reply:
Included https reference in doc string, also adjusted it to fit in 80 columns.